examples/full_screen/src/app.rs

```diff
@@ -9,10 +9,13 @@ pub struct App {
 
 impl App {
     pub fn new(cc: &eframe::CreationContext<'_>) -> Self {
-        let system_shell = std::env::var("SHELL")
-            .expect("SHELL variable is not defined")
-            .to_string();
-
+        let system_shell: String = if cfg!(target_os = "windows") {
```

Comment:
Hi, thanks for your PR, If you would like to change the common code in examples I expect that you will fix all examples, or create the separate example for your case. Many thanks